templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="sidebar-nav">
+  <li>
+    <a href="{% url 'taxi:index' %}">Home page</a>
+  </li>
+  <li>
+    <a href="#">Manufactures</a>
+  </li>
+  <li>
+    <a href="#">Cars</a>
+  </li>
+  <li>
+    <a href="#">Drivers</a>
```

Comment:
The links for 'Manufactures', 'Cars', and 'Drivers' are placeholders (`#`). You should replace these with actual URLs using the `{% url %}` template tag to ensure they point to the correct views in your application.